src/parser/shaman/enhancement/CHANGELOG.tsx

```diff
@@ -6,6 +6,7 @@ import SpellLink from 'common/SpellLink';
 import { change, date } from 'common/changelog';
 
 export default [
+  change(date(2020, 4, 25), <>Converted remaining Enhancement Shaman files to typescript and cleaned its modules.</>, [Vetyst]),
```

Comment:
date is wrong. (Its May not April)

Reply:
This is resolved in PR #3588 which the current changes are based on. I think it will be wiser to have that merged before this PR.